### PR TITLE
Remove last references to notifiers and alerts

### PR DESCRIPTION
**Pull request by JonCrowther**
## Problem
Notifiers and haven't existed in Rancher since 2.7.0, but there are a few lingering references to them in the code.
 
## Solution
Removed the last remnants of notifiers and alerts
 
## Testing
Removed tests that have already been marked as `skips`.

### Automated Testing

* Test types added/modified:
    * Unit
    * Integration (Go Framework)
    * Integration (v2prov Framework)
    * Validation (Go Framework)
    * Other - Explain: _EXPLAIN_
    * None
    * _REMOVE NOT APPLICABLE BULLET POINTS ABOVE_
* If "None" - Reason: _EXPLAIN THE REASON_
  
* If "None" - GH Issue/PR: _LINK TO GH ISSUE/PR TO ADD TESTS_

Summary: _TODO_

### Regressions Considerations

_TODO_

Existing / newly added automated tests that provide evidence there are no regressions:
* _TODO_

### CODE REVIEW

**Review by enrichman — COMMENTED** [review]
I love cleanups, thanks.
LGTM, but some tests are failing. Not sure if related.

I'm approving but lets wait for a green build. :green_circle:

**Review by enrichman — APPROVED** [review]

**Review by alegrey91 — DISMISSED** [review]
LGTM. Let's wait for CI to become green as @enrichman suggested.

File: tests/validation/tests/v1_api/README.md [hunk 1]
```diff
@@ -34,4 +34,3 @@ Examples:
 - test_deployment_with_secret_update()
 - test_monitoring_enable()
 - test_monitoring_update()
```
Comment (diogoasouza) [inline tests/validation/tests/v1_api/README.md:36]: I think we can remove these lines too, no?

Comment (JonCrowther) [reply tests/validation/tests/v1_api/README.md]: Removed!

**Review by diogoasouza — APPROVED** [review]

**Pull request merged**